setupbase.py

```diff
@@ -398,6 +398,28 @@ def run(self):
         update_package_data(self.distribution)
 
 
+def patch_out_bootstrap_bw_print():
+    """Hack! Manually patch out the bootstrap rule that forces printing in B&W.
+
+    We haven't found a way to override this rule with another one.
+    """
+    print_less = pjoin(static, 'components', 'bootstrap', 'less', 'print.less')
+    with open(print_less) as f:
+        lines = f.readlines()
+
+    for ix, line in enumerate(lines):
+        if 'Black prints faster' in line:
+            break
+    else:
+        return  # Already patched out, nothing to do.
+
+    rmed = lines.pop(ix)
+    print("Removed line", ix, "from bootstrap print.less:")
+    print("-", rmed)
+    print()
```

Comment:
why this empty `print` here?

Reply:
It makes a blank line, so the text above is slightly easier to pick out.

Reply:
Fair enough 👍